Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -118,6 +118,32 @@ of removing unused segments after release of internal resources.
   > reflected by queries invoked in subsequent ArangoDB transactions, while
   > in-progress ArangoDB transactions will still continue to return a
   > repeatable-read state.
+  
+  
+  ArangoSearch performs operations in its index based on numerous writer
+  objects that are mapped to processed segments. In order to control memory that
+  is used by these writers (in terms of "writers pool") one can use
+  `writebuffer*` properties of a view.
+
+- **writebufferIdle** (_optional_; type: `integer`; default: `64`;
+  to disable use: `0`)
+
+  Maximum number of writers (segments) cached in the pool.
+
+- **writebufferActive** (_optional_; type: `integer`; default: `0`;
+  to disable use: `0`)
+
+  Maximum number of concurrent active writers (segments) performs (a transaction).
+  Other writers (segments) are wait till current active writers (segments) finish.
+
+- **writebufferSizeMax** (_optional_; type: `integer`; default: `33554432`;
+  to disable use: `0`)
+
+  Maximum memory byte size per writer (segment) before a writer (segment) flush is
+  triggered. `0` value turns off this limit fon any writer (buffer) and will be
+  flushed only after a period [defined](../../Programs/Arangod/Server.md#data-source-flush-synchronization)
+  for the flush thread during ArangoDB server startup. `0` value should be used 
```

Comment:
```suggestion
  [period defined for the flush thread](../../Programs/Arangod/Server.md#data-source-flush-synchronization)
  during ArangoDB server startup. `0` value should be used 
```
Is the meaning that the flush period only be set on server startup, or that flushing only occurs during server startup? We might need to tweak the sentence a bit, to avoid misinterpretation.

Reply:
Fixed.